### PR TITLE
add interceptor

### PR DESCRIPTION
**Pull request by OlegBukalov**

### CODE REVIEW

File: src/app/core/services/toaster.service.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { ToastrService } from 'ngx-toastr';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ToasterService {
+
+  constructor(private toastr: ToastrService) { }
+
+  showSuccess(message: string, title: string){
```
Comment (rmurtazin) [inline src/app/core/services/toaster.service.ts:11]: you need to add the access modificators all over

File: src/app/core/services/auth.interceptor.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  intercept(
+    request: HttpRequest<any>,
+    next: HttpHandler
+  ): Observable<HttpEvent<any>> {
+
+    // add check authentication to AuthService
+    request = request.clone({
+      setParams: {
```
Comment (rmurtazin) [inline src/app/core/services/auth.interceptor.ts:25]: for now we have authService in develop branch you can get token from there or from localhost

File: src/app/core/services/toaster.service.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import { ToastrService } from 'ngx-toastr';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ToasterService {
+
+  constructor(private toastr: ToastrService) { }
+
+  public showSuccess(message: string, title: string){
```
Comment (rmurtazin) [inline src/app/core/services/toaster.service.ts:11]: you need to add type of returned data of all methods all over your in methods

Comment (OlegBukalov) [reply src/app/core/services/toaster.service.ts:11]: toaster methods have no return data

**Review by rmurtazin — APPROVED** [review]

**Pull request merged**